scratch/xbgpu/run-xbgpu.sh

```diff
@@ -24,7 +24,7 @@ tx_affinity=$((affinity + 1))
 tx_comp=$tx_affinity
 other_affinity=$tx_affinity
 src_mcast="239.10.10.$((10 + index)):7148"
-dst_mcast="239.10.11.$((10 + index)):7148"
+send_mcast="239.10.11.$((10 + index)):7148"
```

Comment:
For consistency with the argument names (and with run-fgpu.sh), perhaps call these `src` and `dst`. But whatever you call them, be consistent between the two variables.

Reply:
Too right, my apologies. I think this line caught a stray bullet when updating instances of `dst_`.

Reply:
In this case I think `src` and `dst` are fine - it makes grammatical sense to talk about the source and destination multicast addresses.